src/snapred/backend/dao/state/CalibrantSample/Crystallography.py

```diff
@@ -3,31 +3,24 @@
 
 from pydantic import BaseModel, validator
 
+from snapred.backend.dao.state.CalibrantSample.Atom import Atom
+
 
 class Crystallography(BaseModel):
     """Class containing Crystallography paramaters used in Calibrant Samples
     cif file: string, full path to .cif file containing crystallographic properties
     space group: string, ITOC symbol for the space group
     lattice parameters: List of 6 floats: a, b, c, alpha, beta, gamma
-    atomic parameters:
-
-        atom type: string of chemical symbol
-        atom coordinates:
-
-            x coordinate: float in range [-1, 1]
-            y coordinate: float in range [-1, 1]
-            z coordinate: float in range [-1, 1]
-
-        site occupation factor: float in range [0, 1]
-        adp: positive float, default is 0.01"""
+    atoms: list of atomic parameters for each atom"""
 
     cif_file: str
     space_group: str
     lattice_parameters: List[float]
-    atom_type: str
-    atom_coordinates: List[float]
-    site_occupation_factor: float
-    adp: Optional[float]
+    atoms: List[Atom]
+
+    @property
+    def spaceGroupString(self) -> str:
```

Comment:
This will produce one of the three parameters needed for Mantid's [CrystalStructure](https://github.com/mantidproject/mantid/blob/f59f64c330debd7e3263fa823d1c5fc1f1af706b/Framework/Geometry/inc/MantidGeometry/Crystal/CrystalStructure.h#L73C3-L74C56)
being the third one.
the definition requires 
- unitCellString - > probably maps to lacttice parameters, [supported by this comment](https://github.com/mantidproject/mantid/blob/f59f64c330debd7e3263fa823d1c5fc1f1af706b/Framework/Geometry/inc/MantidGeometry/Crystal/CrystalStructure.h#L43C5-L45C49)
- spaceGroupString     ->  probably maps to space_group      
- scattererString - > the thing this method produces

The additional ask is to make methods that produce these input strings easily, and if you feel like it refactor this to use camelCase.